swan-lake/learn-the-platform/ballerina-tooling/cli-documentation/update-tool.md

```diff
@@ -184,45 +184,49 @@ Use 'bal help dist' for more information on specific commands.
 
 > **Note:** The star (*) indicates the active distribution.
 
-### Pull a specific distribution
+### Pull a distribution
 
-The `bal dist pull <distribution>` command downloads a particular distribution and stores it in your local environment. It also sets the fetched distribution as the active distribution.
+You can pull and update to a preferred (latest or any) Ballerina distribution as follows. 
+
+#### Pull the latest distribution
```

Comment:
Shall we change order by mentioning specific distribution first and then mention instead  version you can mention `latest`

Reply:
Done.